### PR TITLE
feat: added Cart Management page

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:
#Description: Added Cart management
cartManagement
 page
#Screenshot:

### CODE REVIEW

**Review by masraniJay — CHANGES_REQUESTED** [review]
review1
Button is outside the card.
review2
it would look good if there'd be some space here.

File: components/cartPage/cart.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                            <span><button class="log"><a href="/components/Login/login.html" target="_blank" rel="noopener noreferrer">Log In</a></button></span>
+                            <span><a href="/components/wishlistPage/wishlist.html" target="_blank"
+                                rel="noopener noreferrer"><i class="fas fa-2x fa-heart"></i></a></span>
+                            <span><a href="/components/cartPage/cart.html" target="_blank" rel="noopener noreferrer"><i class="fas fa-2x fa-shopping-cart"></i></a></span>
+
+                        </div>
+                    </div>
+
+                    <div class="cart">
+                        <h1 class="cart-Heading"><span>My Cart</span><span>(1)</span></h1>
```
Comment (ooreally) [inline components/cartPage/cart.html:63]: Use a gutter before heading for better UI

File: components/cartPage/cart.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                </div>
+                <div class="sidebar__item">
+                    <a href="/"><i class="fas fa-trash"><span class="textDrawer">Trash</span></i></a>
+                </div>
+                <div class="sidebar__item">
+                    <a href="/"><i class="fas fa-exclamation-triangle"><span class="textDrawer">Spam</span></i></a>
+                </div>
+            </div>
+            <div class="mainBody">
+                <div class="modal__body" id="modalDrawerMainBody">
```
Comment (ooreally) [inline components/cartPage/cart.html:47]: This is not taking the whole width of viewport. There is some white space left on the right.

**Review by ooreally — COMMENTED** [review]
Overall page looks good. Suggestions:
- change width of modal body to whole viewport
- space should be there between heading and header

**Pull request merged**